services/basic_types.proto

```diff
@@ -474,13 +478,12 @@ enum TokenPauseStatus {
  *
  * A Key can contain a ThresholdKey or KeyList, which in turn contain a Key, so this mutual
  * recursion would allow nesting arbitrarily deep. A ThresholdKey which contains a list of primitive
- * keys (e.g., ed25519) has 3 levels: ThresholdKey -> KeyList -> Key. A KeyList which contains
- * several primitive keys (e.g., ed25519) has 2 levels: KeyList -> Key. A Key with 2 levels of
+ * keys (e.g., Ed25519) has 3 levels: ThresholdKey -> KeyList -> Key. A KeyList which contains
+ * several primitive keys (e.g., Ed25519) has 2 levels: KeyList -> Key. A Key with 2 levels of
```

Comment:
Here and in the sentence before we should either remove `(e.g., Ed25519)` and just reference our previous definition of primitive keys (preferred) _or_ add `ECDSA(secp256k1)` to these lists.

Reply:
Removed the  `(e.g., Ed25519)`  and just used `primitive keys` . Thanks @steven-sheehy